packages/execution-environments/src/common/commands.ts

```diff
@@ -25,19 +25,38 @@ export type CommandMethodsMapping = {
  * @param request - The request object.
  * @returns The formatted arguments.
  */
+// eslint-disable-next-line consistent-return
```

Comment:
Weird, that shouldn't be necessary. Maybe `return assertExhaustive()` will solve it?